### PR TITLE
More style changes, delay, and content type

### PR DESCRIPTION
**Pull request by davidkpiano**
Addressing comments here: https://github.com/Glimpse/Glimpse.Client.Hud/pull/109#issuecomment-307890492

> Looking great! Looks like content type is missing? Does the URL, status test and content type (when added) overflow correctly. Also if we could add a little more space between request items in the ajax list and could we get a little more margin space to the left of the dividing line... thinking about it having a bit more of a sense of space. Also are you planning on adding the mouse over delay in this PR?

jun-19-2017 18-13-25

### CODE REVIEW

File: src/index.scss [hunk 1]
```diff
@@ -8,6 +8,7 @@ $type-scales: (
     1: 18px,
 );
 $border-height: 3px;
+$popup-delay: 1s;
```
Comment (avanderhoorn) [inline src/index.scss]: Can we try reducing this to 0.25? I'm aiming to it just be enough that a use can move their mouse down and over it (as if they where going to the windows/mac task bar) without it opening, but fast enough that if they slow down or pause at all it feels like its opening? And the same should be for mouse off, if you mouse goes off and comes straight back on, it shouldn't start to close. Best advise I can give it to mirror the behavior in this regards (in terms of feel) from the old hud on http://play.getglimpse.com/.

**Comment by avanderhoorn** [comment]
I can't see the following from the screen shot:
 - Do things overflow correctly?
 - Could we get a little more margin space to the left of the dividing line?

Also can the "primary" number be emphasized (made bigger by 20%) in the popup... i.e. the Page Load Time and Ajax Requests number.

**Comment by davidkpiano** [comment]
Updated screenshots:
screen shot 2017-06-20 at 4 25 45 pm

**Comment by davidkpiano** [comment]
Updated screens:
screen shot 2017-06-20 at 4 25 45 pm
screen shot 2017-06-20 at 4 25 39 pm

**Comment by avanderhoorn** [comment]
Looks good. The increased font size should only apply to the "Load Time" and "Ajax Requests" values, not the sub timing values. If they could go back to what they were what would be great. Also we can probably decrease the gap between the method and the status code... most methods are fairly short. Maybe remove 20px?

**Comment by davidkpiano** [comment]
Okay, I made the width of that column to `8ch` specifically to handle `DELETE` and `OPTIONS`, as (for most cases) that's probably the longest methods:
screen shot 2017-06-20 at 6 38 38 pm
